### PR TITLE
issue#081-conditional-unlock-user-update-endpoint-access

### PR DESCRIPTION
**Pull request by student0b4dc0d3**
completed the requirements of ticket #81 
please review and comment / resolve
thank you

### CODE REVIEW

**Comment by student0b4dc0d3** [comment]
Made a small correction to improve functionality
Please review and comment

Thank you

File: backend/src/routes.js [hunk 1]
```diff
@@ -320,4 +333,12 @@ function getDocumentState(queryState) {
...
 }
 
-module.exports = router;
\ No newline at end of file
+// issue081 unlock user-update endpoint
+// here: implement helper function
+function removeObjectProperties(originalObject={},propNamesArray=[]){
+    const result = JSON.parse(JSON.stringify(originalObject));
+    for (i of propNamesArray) delete result[i];
+    return result;
```
Comment (Stef-Lev) [inline backend/src/routes.js:341]: I think if you add a let variable here would make the loop safer.

File: backend/src/routes.js [hunk 2]
```diff
@@ -320,4 +333,12 @@ function getDocumentState(queryState) {
...
+};
+
```
Comment (Stef-Lev) [inline backend/src/routes.js:343]: I think if you add a let variable here would make the loop safer.

**Comment by student0b4dc0d3** [comment]
Included suggested modification by @Stef-Lev 
Please review and comment
Thank you

**Pull request merged**